tests/cross/src/test/scala/tests/pc/InsertInferredTypeSuite.scala

```diff
@@ -31,6 +31,77 @@ class InsertInferredTypeSuite extends BaseCodeActionSuite {
        |}""".stripMargin,
   )
 
+  checkEdit(
+    "wrong-val",
+    """|object A{
+       |  val <<alpha>> :  String = 123
```

Comment:
```suggestion
       |  val <<alpha>>: String = 123
```
I'm curious does this work?

Reply:
It does, actually I also tested that in the LSP tests. I also changed the example here.